docs(trace-and-optimize): fix about Merge Skinned Mesh (#1031)
anatawa12/AvatarOptimizer: .docs/content/docs/reference/trace-and-optimize/index.md

- `Optimize Animator`\
  Optimizes Animator Controller. See [this section](#animator-optimizer) for more details.
- `Merge Skinned Meshes`\
  Merges skinned meshes which don't need to be separated.\

>   Some meshes may not be automatically merged depending on their condition, so if necessary, use [Merge Skinned Mesh](../merge-skinned-mesh).

Reviewer: ```suggestion
  Some meshes may not be automatically merged in some cases, so use [Merge Skinned Mesh](../merge-skinned-mesh) manually if necessary.
```
こうしてみるのはどうでしょうか?

Author: いい感じに修正ありがとうございます 🙇
